ext_conf_template.txt

```diff
@@ -1,10 +1,8 @@
 # cat=Settings/O; type=string; label= Deepl API Key
 apiKey =
-# cat=Settings/O; type=string; label= Deepl API Url
+# cat=Settings/O; type=string; label= Deepl API Url: Please note that with Deepl Free, the API host is https://api-free.deepl.com
 apiUrl = https://api.deepl.com/v2/translate
-# cat=Settings/O; type=string; label= Google translate API Key(Non freemode)
+# cat=Settings/O; type=string; label= Google Translate API Key (Non freemode)
 googleapiKey =
-# cat=Settings/O; type=string; label= Google translate API Url(Non freemode)
-googleapiUrl = https://translation.googleapis.com/language/translate/v2
```

Comment:
I can't really see why a diff shows a change for this line (googleapiUrl) but the rest looks good :smile:

Reply:
Remove unneccessary empty lines :-)